superset/dashboards/filters.py

```diff
@@ -189,12 +189,14 @@ def apply(self, query: Query, value: Any) -> Query:
             return query.filter(
                 and_(
                     Dashboard.certified_by.isnot(None),
+                    Dashboard.certified_by != '',
                 )
             )
         if value is False:
             return query.filter(
-                and_(
+                or_(
                     Dashboard.certified_by.is_(None),
+                    Dashboard.certified_by == '',
```

Comment:
```suggestion
                    Dashboard.certified_by == "",
```
These two changes should fix the `black` linting issues the PR is facing. 

FWIW, I looked a the CI logs, which show the file failing linting. You can then run `black superset/dashboards/filters.py` to fix/commit these changes.